src/main/java/org/ow2/proactive/catalog/rest/entity/CatalogObjectRevisionEntity.java

```diff
@@ -215,14 +215,14 @@ public void setRawObject(byte[] rawObject) {
 
     @Override
     public String toString() {
-        return "CatalogObjectRevision{" + "commitId=" + commitId + ", kind='" + kind + '\'' + ", commitMessage='" +
-               commitMessage + '\'' + ", commitDate=" + commitDate + ", name='" + name + '\'' + ", bucketId=" +
-               bucketId + ", contentType='" + contentType + '\'' + ", keyValueMetadataList=" + keyValueMetadataList +
-               '}';
+        return "CatalogObjectRevisionEntity{" + "commitId=" + commitId + ", kind='" + kind + '\'' +
```

Comment:
if we use Objects.toString,  or apache common ToStringBuilder library to construct toString method, it will be much easier